src/components/Header.tsx

```diff
@@ -1,12 +1,32 @@
-import { Grid, Typography } from "@mui/material";
+import { Box, Collapse, Grid, Typography } from "@mui/material";
+import { useEffect, useState } from "react";
 import { ArtistProfile } from "./ArtistProfile";
 import { NEWMLogo } from "./NEWMLogo";
 import { Artist } from "../models/artist";
 
 interface HeaderProps {
   artist: Artist;
 }
+
 export const Header = ({ artist }: HeaderProps) => {
+  const [isTextVisible, setIsTextVisible] = useState(true);
+
+  useEffect(() => {
+    const listenToScroll = () => {
+      const heightToHideFrom = 37;
+      const winScroll = document.getElementById("content")?.scrollTop;
+  
+      if (winScroll && winScroll > heightToHideFrom) {
+        isTextVisible && setIsTextVisible(false); // to limit setting state only the first time
+      } else {
+        setIsTextVisible(true);
+      }
+    };
+    document && document.getElementById("content")?.addEventListener("scroll", listenToScroll);
```

Comment:
I would add some logs around these calls to make sure your scroll listener is getting attached